lib/plugins/create/create.js

```diff
@@ -46,7 +33,79 @@ class Create {
 
   async create() {
     if ('template' in this.options) {
-      return this.createFromTemplate();
+      mainProgress.notice('Setting up new project', { isMainEvent: true });
+
+      if (!recommendedTemplatesList.includes(this.options.template)) {
```

Comment:
Yes - I've made such decision in order to make it faster to check if a template exists or not for this command - otherwise we have to fetch all templates to say that template X is not available - we also do not intend to add new such templates to legacy folder. The lists are equivalent at the moment - if a template is in `legacy` folder in examples, it means it's listed in `recommended-list/index.js` file. Did you found that to not be the case?

Reply:
> if a template is in legacy folder in examples, it means it's listed in recommended-list/index.js file. Did you found that to not be the case?

By quickly looking I've assumed it was the case, but that was the wrong impression (in the recommended list they're not ordered alphabetically but by provider). Therefore please ignore that comment. Sorry for confusiun